src/resolvers/Query/helperFunctions/getInputArgs.ts

```diff
@@ -0,0 +1,713 @@
+  if (where.appLanguageCode_not) {
+    inputArgsPayload = {
+      ...inputArgsPayload,
+      appLanguageCode: {
+        $ne: where.appLanguageCode_not,
+      },
+    };
+  }
+
+  // Objects appLanguageCode falls in provided list
+  if (where.appLanguageCode_in) {
+    inputArgsPayload = {
+      ...inputArgsPayload,
+      appLanguageCode: {
+        $in: where.appLanguageCode_in,
+      },
+    };
+  }
+
+  // Return objects appLanguageCode not falls in the list
+  if (where.appLanguageCode_not_in) {
+    inputArgsPayload = {
+      ...inputArgsPayload,
+      appLanguageCode: {
+        $nin: where.appLanguageCode_not_in,
+      },
+    };
+  }
+
+  // Return objects with appLanguageCode containing provided string
+  if (where.appLanguageCode_contains) {
+    inputArgsPayload = {
+      ...inputArgsPayload,
+      appLanguageCode: {
+        $regex: where.appLanguageCode_contains,
+        $options: "i",
+      },
+    };
+  }
+
+  // Returns objects with appLanguageCode starts with provided string
+  if (where.appLanguageCode_starts_with) {
+    const regexp = new RegExp("^" + where.appLanguageCode_starts_with);
+    inputArgsPayload = {
+      ...inputArgsPayload,
+      appLanguageCode: regexp,
+    };
+  }
+
+  // Return users with admin for provided organizationId
+  if (where.admin_for) {
+    inputArgsPayload = {
+      ...inputArgsPayload,
+      adminFor: {
+        _id: where.admin_for,
+      },
+    };
+  }
+
+  if (where.event_title_contains) {
+    inputArgsPayload = {
+      ...inputArgsPayload,
+      "registeredEvents.title": {
+        $regex: where.event_title_contains,
+        $options: "i",
+      },
+    };
+  }
+
+  //Returns provided text objects
+  if (where.text) {
+    inputArgsPayload = {
+      ...inputArgsPayload,
+      text: where.text,
+    };
+  }
+
+  //Returns objects with not the provided text
+  if (where.text_not) {
+    inputArgsPayload = {
+      ...inputArgsPayload,
+      text: {
+        $ne: where.text_not,
+      },
+    };
+  }
+
+  //Return objects with the given list text
+  if (where.text_in) {
+    inputArgsPayload = {
+      ...inputArgsPayload,
+      text: {
+        $in: where.text_in,
+      },
+    };
+  }
+
+  //Returns objects with text not in the provided list
+  if (where.text_not_in) {
+    inputArgsPayload = {
+      ...inputArgsPayload,
+      text: {
+        $nin: where.text_not_in,
+      },
+    };
+  }
+
+  //Returns objects with text containing provided string
+  if (where.text_contains) {
+    inputArgsPayload = {
+      ...inputArgsPayload,
+      text: {
+        $regex: where.text_contains,
+        $options: "i",
+      },
+    };
+  }
+
+  //Returns objects with text starts with that provided string
+  if (where.text_starts_with) {
+    const regexp = new RegExp("^" + where.text_starts_with);
+    inputArgsPayload = {
+      ...inputArgsPayload,
+      text: regexp,
+    };
+  }
+
+  return inputArgsPayload as T;
```

Comment:
`as T` no longer needed after the changes above.
```ts
return inputArgsPayload;
```

Reply:
Done